db/migrate/20220427004448_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly add the necessary foreign keys to the `Comment` table which will create `author_id`, `post_id`, and a corresponding index. The foreign key is on the ERD diagram.